Production_scripts/Process_extracts/extract_maternity.R

```diff
@@ -166,7 +166,9 @@ maternity_clean <- maternity_file %>%
     discondition = factor(discondition,
       levels = c(1:5, 8)
     )
-  )
+  ) %>%
+  # Add SMR type
+  mutate(smrtype = add_smr_type(recid, mpat))
```

Comment:
```suggestion
  ),
  # Add SMR type
  smrtype = add_smr_type(recid, mpat)
  )
```
Wrap mutate together